relay_api.go

```diff
@@ -20,7 +20,9 @@
 
 package tchannel
 
-import "github.com/uber/tchannel-go/relay"
+import (
+	"github.com/uber/tchannel-go/relay"
+)
```

Comment:
nit: can revert this change

Reply:
not sure why? we're passing in `relay.RespFrame` below

Reply:
Sorry, what I meant is that the import matches the same as what it was previously, but it's grouped (which isn't necessary since there's only one import)